api/envoy/service/ext_proc/v3/external_processor.proto

```diff
@@ -124,6 +138,9 @@ message ProcessingRequest {
   //   are needed.
   //
   bool observability_mode = 10;
+
+  // The filter configuration to be sent to the server.
+  FilterConfigToSend filter_config = 11;
```

Comment:
Consider renaming `FilterConfigToSend` to `FilterConfigContext`, as not the entire config is sent, but some filter config context.

Reply:
`ProtocolConfiguration` would be even better.

I think also the ext_proc server only needs to know if body streaming is full or half duplex. It does not need all other options (buffered, etc). I think we should just limit to two cases, full duplex or half duplex. The default value in this case would be half-duplex, if it is old ext_proc filter that does not set this option.

Reply:
Done. Also encoded send_body_without_waiting_for_header_response flag